WebApp/src/components/BasicTest/BasicTestContainer.css

```diff
@@ -0,0 +1,11 @@
+.container {
```

Comment:
Peut-être pas le but de cette PR mais on devrais utiliser [styled-components](https://styled-components.com/) pour gérer notre css. Styled components est probablement le plus populaire mais il y en a d'autre du genre.

Reply:
Added to backlog!